app/presenters/two_factor_login_options_presenter.rb

```diff
@@ -3,13 +3,12 @@ class TwoFactorLoginOptionsPresenter < TwoFactorAuthCode::GenericDeliveryPresent
 
   attr_reader :current_user
 
-  def initialize(current_user, view, service_provider, aal3_policy)
+  def initialize(current_user, view, service_provider, service_provider_mfa_policy)
     @current_user = current_user
     @view = view
     @service_provider = service_provider
-    return unless aal3_policy
-    @aal3_required = aal3_policy.aal3_required?
-    @piv_cac_required = aal3_policy.piv_cac_required?
+    @aal3_required = service_provider_mfa_policy.aal3_required?
+    @piv_cac_required = service_provider_mfa_policy.piv_cac_required?
```

Comment:
if these are the only parts we need from the policy, should we just add params for `aal3_required` and `piv_cac_required` instead?

Reply:
Yeah, let me see how that works out